instrumentation/jsonrpc4j-1.3/library/src/main/java/io/opentelemetry/instrumentation/jsonrpc4j/v1_3/JsonRpcTelemetryBuilder.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jsonrpc4j.v1_3;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.incubator.semconv.rpc.RpcClientAttributesExtractor;
+import io.opentelemetry.instrumentation.api.incubator.semconv.rpc.RpcClientMetrics;
+import io.opentelemetry.instrumentation.api.incubator.semconv.rpc.RpcServerAttributesExtractor;
+import io.opentelemetry.instrumentation.api.incubator.semconv.rpc.RpcServerMetrics;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.InstrumenterBuilder;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JsonRpcTelemetryBuilder {
+
+  private static final String INSTRUMENTATION_NAME = "io.opentelemetry.jsonrpc4j-1.3";
+
+  private final OpenTelemetry openTelemetry;
+
+  private final List<
+          AttributesExtractor<? super JsonRpcClientRequest, ? super JsonRpcClientResponse>>
+      additionalClientExtractors = new ArrayList<>();
+  private final List<
+          AttributesExtractor<? super JsonRpcServerRequest, ? super JsonRpcServerResponse>>
+      additionalServerExtractors = new ArrayList<>();
+
+  JsonRpcTelemetryBuilder(OpenTelemetry openTelemetry) {
+    this.openTelemetry = openTelemetry;
+  }
+
+  /**
+   * Adds an extra client-only {@link AttributesExtractor} to invoke to set attributes to
+   * instrumented items. The {@link AttributesExtractor} will be executed after all default
+   * extractors.
+   */
+  @CanIgnoreReturnValue
+  public JsonRpcTelemetryBuilder addClientAttributeExtractor(
```

Comment:
since you really offer library instrumentation only for the server you shouldn't have anything related to the client instrumentation here

Reply:
moved all client stuff to `javaagent` package